<!DOCTYPE HTML>

<html>
	<head>
		<title>Raehash Shah</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Header -->
			<header id="header">
				<div class="inner">
					<a href="#" class="image avatar"><img src="images/Headshot_short.jpeg" alt="" /></a>
					<h1><strong>Hi, my name is Raehash Shah!</strong><br />
					Carnegie Mellon Masters Student <br />
					studying Computational Biology.</h1>
				</div>
			</header>

		<!-- Main -->
			<div id="main">

				<!-- One -->
					<section id="one">
						<header class="major">
							<h2>Welcome to my Website!</h2>
						</header>
						<p>I am an enthusiastic student actively exploring opportunities in the biotechnology industry, with a keen interest in drug discovery for diseases. 
							My expertise lies in utilizing machine learning techniques to interpret and analyze biomedical data. Below are some of my most significant projects, 
							showcasing my diverse experiences and skills I've developed in biotechnology.</p>
						<ul class="actions">
							<li><a href="Raehash_Shah_Resume_2025.pdf" class="button">Resume</a></li>
						</ul>
					</section>

				<!-- Two -->
					<section id="two">
						<h2>Recent Work</h2>
						<div class="row">
							<article class="col-6 col-12-xsmall work-item">
								<a href="images/Projects/NLP_Pubmed.jpg" class="image fit thumb"><img src="images/Projects/NLP_Pubmed.jpg" alt="" /></a>
								<h3>PubMed Search Engine (May - Aug 2024)</h3>
								<p>As an intern at GEn1E Lifesciences, I developed a PubMed search engine which aids researchers to review published papers to gain insights into 
									ongoing work in the field of drug discovery that gathers more detailed information than standard PubMed searches. Therefore using Large Language 
									Models (LLMs) and Knowledge Graphs, the tool I developed can enhance our understanding of relevant studies that aid in advancing precision therapies.</p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="images/Projects/LA_Scores.png" class="image fit thumb"><img src="images/Projects/LA_Scores.png" alt="" /></a>
								<h3>Generating COVID-19 Sequencing Data Through a Graphical Representation of DNA Sequence (Apr - May 2024) </h3>
								<p>In a class project at Carnegie Mellon University, we applied graphical methods discussed in our course to see if these tools can produce plausible mutated sequences of 
									COVID-19. We specifically assessed Hidden Markov Models, Variational Autoencoders, and Generative Adversarial Networks, to evaluate their effectiveness. To measure the 
									success of the methods, which we compared the GC content of our generated sequences and performed local alignment against a consensus COVID-19 genome. This dual approach 
									not only tests the structural accuracy of our models but also their practical applicability in simulating the evolutionary dynamics of the virus. <a href="https://github.com/cosmic20/10708Project"> Learn More.</a></p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="images/Projects/SIMON.jpg" class="image fit thumb"><img src="images/Projects/SIMON.jpg" alt="" /></a>
								<h3>SIMON - SIgnal siMulated by diffusiON (May - Aug 2023) </h3>
								<p>SIMON is a conditional diffusion based machine learning model which generates large amounts of ONT signal data from a nucleotide sequence. This tool 
									will help provide adequate amounts of data for the QC Virology Development team at Regeneron to generate tools to help supplement current detection 
									protocols for viruses. Currently SIMON has been shown to perform well in generating sequencing data for CHO (mammalian) sequencing data. <a href="https://github.com/Shahraehash/Regeneron_SIMON"> Learn More.</a></p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="images/Projects/Sniffer.jpg" class="image fit thumb"><img src="images/Projects/Sniffer.jpg" alt="" /></a>
								<h3>TDM Protocol Analyzer (May - Aug 2022)</h3>
								<p>At Werfen Instrumentation Laboratory, Beacon, a throughput hemostasis testing device, there are multiple L3 boards which control varying number of L4 boards. 
									The communication between the L3 host and up to 7 L4 nodes is across a RS-485 bus which is built on a time-division multiplexing (TDM) principle. This means 
									there are packets of data that is exchanged between the L3 board and L4 boards regularly. Each cycle consists of a L3 board sending a fixed packet to specific 
									defined nodes and those nodes later respond with a status packet. This sniffer's goal is to analyze each of the packets that are sent across this bus and report it to the user. <a href="https://github.com/Shahraehash/Werfen_TDMProtocolAnalyzerSniffer"> Learn More.</a></p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="images/Projects/Modeling.jpg" class="image fit thumb"><img src="images/Projects/Modeling.jpg" alt="" /></a>
								<h3>Comparing Temporal Change of DNA Methylation Between Type 1 Diabetes Patients and Healthy Individual (Nov - Dec 2021)</h3>
								<p>In a class project at Carnegie Mellon University, we trained three Markov Models to model the change of methylation sites
									with respect to age in individuals with type 1 diabetes and healthy individuals.
									We prune and structurally compare Markov Models to identify distinct patterns
									of methylation in healthy individuals and those with type 1 diabetes. Moreover,
									we show these Markov Models are generalizable by using them to predict age
									given unseen methylation sites and benchmark their performance against a linear
									regression baseline and deep neural network. <a href="https://drive.google.com/file/d/1ZKoJ3_Nztl6clYDGWTkw9sHyW-hoFo1A/view?usp=sharing"> Learn More.</a></p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="images/Projects/Organoids.jpg" class="image fit thumb"><img src="images/Projects/Organoids.jpg" alt="" /></a>
								<h3>Analysis of RNA-Seq from Organoids to Express Limited Tumor Heterogeneity (Jul 2019 - Aug 2021)</h3>
								<p>I worked with organoids in the Englander Institute for Precision Medicine to compare gene expression from Primary Tumor Samples to different stages in the organoid cell culture 
									(i.e. Passage 5 and Passage 10). Through Hierarchical Clustering, Principal Component Analysis (PCA) and differential gene expression analysis and clustering analysis, the results 
									demonstrated to what extent the organoids maintained their characteristics to a live tumor sample. The results suggested very little difference in some organoids and their matching 
									patient tumors. However, as tumors were cultured to organoids, the organoids clustered well from passage 5 to passage 10. <a href="https://drive.google.com/file/d/1A0g58V0B9iu8p1X9WwSl4asImdC6X7sd/view?usp=sharing"> Learn More.</a></p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="images/Projects/RHD.jpg" class="image fit thumb"><img src="images/Projects/RHD.jpg" alt="" /></a>
								<h3>The Role of Glycoproteins and HLA Markers in the Fibrosis of Rheumatic Valvular Disease (Sep 2018 - Jun 2020)</h3>
								<p>I investigated how the impact of streptolysin treated T cells could influence the “fibrosis” process of differentiated myofibroblasts as evidenced by the production of EDA-fibronectin 
									and proliferation. CD4+, CD8+, and NK-92 T cell subsets were treated with streptolysin to determine the production of IFN-y, IL-6, and IL-17A. In addition, the secretome produced by 
									the streptolysin treated subsets of these T cells was added to the differentiated myofibroblasts to observe the “fibrosis” process. To examine the impact of contact with the myofibroblasts, 
									CD4+ and CD8+ T cells were cultured together with the myofibroblasts, as well as individually. In addition, HLA genes encoding MHC complexes were analyzed using 
									bioinformatics to determine if specific HLA types have a role in rheumatic heart disease. These MHC complexes bind to the T cell receptors, eliciting autoreactivity of the T cell, 
									facilitating fibrosis of the mitral valve. This project was performed at Bergen County Academies. <a href="https://drive.google.com/file/d/1oF4Oo4nSmBwlt1OLBzKCs9xdXLIPGvRb/view?usp=sharing"> Learn More.</a></p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="images/Projects/Pittsburgh_Rivers.jpg" class="image fit thumb"><img src="images/Projects/Pittsburgh_Rivers.jpg" alt="" /></a>
								<h3>Visualization of the Species in the Pittsburgh Rivers (Jul - Aug 2019)</h3>
								<p>Extracted and Sequenced the DNA from the water samples through bacterial culture, PCR, Sanger sequencing and Nextgen sequencing. 
									Computationally determined what species were present in the water samples. Co-Author on the publication and the AKPR sequencing 
									of the Flectobacillus roseus bacteria. From the DNA Samples developed UPGMA trees, Bar Plots, Unifrac Distance Matrices and PCoA plots. 
									This project was conducted at Carnegie Mellon University.<a href="https://drive.google.com/file/d/16Grk3ETi9viURhDWD9_ebK95jtqmWVxZ/view?usp=sharing"> Learn More.</a></p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="images/Projects/Virome_Sniff.jpg" class="image fit thumb"><img src="images/Projects/Virome_Sniff.jpg" alt="" /></a>
								<h3>Virome Sniff (Jun 2017)</h3>
								<p>Developed a tool that would detect and visualize from NGS data. The tool would search for viruses in protein, rather than nucleotide 
									space to capture and characterize larger number of viruses and detect viruses- associated domains in the sample. My specific role within the project was to develop
									the visualization and the user interface for the project. This project was performed over a few days at the New York Genome Center Hackathon.  <a href="https://github.com/NCBI-Hackathons/Virus_Domains"> Learn More.</a></p>
							</article>
						</div>
						<!--
						<ul class="actions">
							<li><a href="#" class="button">Full Portfolio</a></li>
						</ul>-->
					</section>

				<!-- Three -->
					<section id="three">
						<h2>Get In Touch</h2>
						<p>I’m always open to discussing new projects or any opportunities within the biotechnology and drug discovery fields. 
							Whether you have a question, want to connect, or explore potential partnerships, feel free to reach out.</p>
						<div class="row">
							<!--
							<div class="col-8 col-12-small">
								<form method="post" action="#">
									<div class="row gtr-uniform gtr-50">
										<div class="col-6 col-12-xsmall"><input type="text" name="name" id="name" placeholder="Name" /></div>
										<div class="col-6 col-12-xsmall"><input type="email" name="email" id="email" placeholder="Email" /></div>
										<div class="col-12"><textarea name="message" id="message" placeholder="Message" rows="4"></textarea></div>
									</div>
								</form>
								<ul class="actions">
									<li><input type="submit" value="Send Message" /></li>
								</ul>
							</div>
							-->
							<div class="col-4 col-12-small">
								<ul class="labeled-icons">
									<li>
										
									</li>
									<li>
										<h3 class="icon solid fa-mobile-alt"><span class="label">Phone</span></h3>
										<a>(201) 449-4918</a>
									</li>
									<li>
										<h3 class="icon solid fa-envelope"><span class="label">Email</span></h3>
										<a>raehashshah@gmail.com</a>
									</li>
								</ul>
							</div>
						</div>
					</section>


			</div>

		<!-- Footer -->
			<footer id="footer">
				<div class="inner">
					<ul class="icons">
						<li><a href="https://x.com/RaehashS" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://github.com/Shahraehash" class="icon brands fa-github"><span class="label">Github</span></a></li>
						<li><a href="https://www.linkedin.com/in/raehash-shah/" class="icon solid fa-users"><span class="label">LinkedIn</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; Untitled</li><li>Design: <a>HTML5 UP</a></li>
					</ul>
				</div>
			</footer>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.poptrox.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>